### ISSUE TITLE
Update Mermaid.js

### ISSUE THREAD

**Issue by MrDj200** [secure-77/Perlite#95]
**Is your feature request related to a problem? Please describe.**
Mermaid.js is outdated and does not render my graph. Instead it's giving me a "Syntax Error"
![image](https://github.com/secure-77/Perlite/assets/5685500/da39dcff-2574-4ad4-9120-aa82976404c9)

**Describe the solution you'd like**
I'd like mermaid.js to be updated to a newer version.
Perlite seems to be quite a few major releases behind

**Describe alternatives you've considered**
I considered using Canvas, but that is not supported at all by Perlite at the time of writing (#72)

**Additional context**
The mermaid chart I'm trying to show:
```
---
title: "Arma 3 - Basic Medical in ACE"
---
flowchart TD
    977290(["Unresponsive patient"]) --> 289944{"Is their body ragdolled or limp?"}
    289944 -->|"Yes"| 227197("Patient ded")
    289944 -->|"No"| 335680{"Lost more than #quot;some#quot; blood?"}
    335680 -->|"No"| 122890("Inject Epinephrine")
    335680 -->|"Yes"| 623814("Stop the bleeding.\nRefer to Bleeding guide")
    122890 --> 623814
    623814 --> 549744("Transfuse blood.\nRefer to  transfusion guide")
    549744 --> 682444("Fully bandage all limbs")
    682444 --> 109205("Remove tourniquets")
    109205 --> 263334{"Lost more than #quot;some#quot; blood?"}
    263334 -->|"Yes"| 644629("Wait for transfusion")
    644629 --> 263334
    263334 -->|"No"| 432937{"Patient still unconscious?"}
    432937 -->|"Yes"| 829784("Inject Epinephrine")
    432937 -->|"No"| 601555(["Alive!"])
```

**Comment by secure-77** [comment]
this is updated on the dev branche, will merge it in the next days to main

**Comment by secure-77** [comment]
fixed in 1.5.7, mermaid is now loaded from the CDN
